ext/standard/password.c

```diff
@@ -83,7 +83,7 @@ static zend_string* php_password_make_salt(size_t length) /* {{{ */
 	zend_string *ret, *buffer;
 
 	if (length > (INT_MAX / 3)) {
-		php_error_docref(NULL, E_WARNING, "Length is too large to safely generate");
+		zend_value_error("Length is too large to safely generate");
 		return NULL;
```

Comment:
I think the convention is to ``return;`` after an Error/Exception if could please change those too.

Reply:
Yes, you are right, but I wasn't sure because this piece of code is not located in a `ZEND_FUNCTION()`.

Reply:
Ah mb didn't look at the code flow I thought it was in a PHP_FUNCTION